### PR TITLE
fixed footer to the bottom of the page

### PR DESCRIPTION
**Pull request by justinavickers**

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -59,11 +59,15 @@ h1{
 footer {
     background-color: #6C7A89;
     border-radius: 10px;
+    position: absolute; bottom: 0;
+    margin-bottom: 10px;
+    min-width: 99%;
 }
 footer ul {
+    display: flex;
```
Comment (naienko) [inline main.css:67]: this line is in the css twice

File: main.css [hunk 2]
```diff
@@ -59,11 +59,15 @@ h1{
...
+    justify-content: space-around;
```
Comment (naienko) [inline main.css]: I thought we had preferred the look give by space-evenly

**Comment by naienko** [comment]
please see review comments

File: career.html [hunk 3]
```diff
@@ -40,7 +40,12 @@ <h1>Grace Hopper's Career</h1>
       research fellow under a Navy contract at Harvard.[19]
  </div>
 
-  </body>
-<script src="index.html"></script>
-
+ <footer>
```
Comment (naienko) [inline career.html]: test

**Comment by kellycoles** [comment]
👍

**Pull request merged**